snuba/manual_jobs/runner.py

```
@@ -134,10 +134,11 @@ def run_job(job_spec: JobSpec) -> JobStatus:
         if not job_spec.is_async:
             current_job_status = _set_job_status(job_spec.job_id, JobStatus.FINISHED)
             job_logger.info("[runner] job execution finished")
-    except BaseException as e:
+    except Exception as e:
```

Why did this exception type need change? AFAIK, `BaseException` is used to catch all exceptions including the ones needed to terminate the program, while `Exception` is used to catch exceptions that do not terminate the program.

---

My mistake, fixed!